3-variant_qc/7-plot_rf_output.py

```diff
@@ -592,6 +594,7 @@ def main():
     plot_dir = os.path.join(root_plot_dir, "variant_qc", model_id)
 
     # = STEP LOGIC = #
+    hail_utils.init_hl(tmp_dir)
```

Comment:
why adding it here?

Reply:
The function create_plots() reads Hail tables, and it triggers the implicit init of Hail. I decided that it's better to init Hail explicitely.